vivid-server/src/modules/friends/friends.service.ts

```diff
@@ -28,10 +28,10 @@ export class FriendsService {
 
   // Add FriendEntity to database (=> send friend request)
   async sendFriendRequest(
-    user_1: string,
-    user_2: string,
-    requested_by: string,
-    requested_to: string,
+    user_1: UserEntity,
+    user_2: UserEntity,
```

Comment:
dont use entities here, they are hard to get and you only need the ids anyway
(also this break the comparison check on line 37)